pkg/scheduler/framework/plugins/nodeaffinity/node_affinity_test.go

```diff
@@ -1021,8 +1023,11 @@ func TestNodeAffinityPriority(t *testing.T) {
 				{ObjectMeta: metav1.ObjectMeta{Name: "node2", Labels: label2}},
 				{ObjectMeta: metav1.ObjectMeta{Name: "node3", Labels: label3}},
 			},
-			expectedList: []framework.NodeScore{{Name: "node1", Score: 0}, {Name: "node2", Score: 0}, {Name: "node3", Score: 0}},
-			runPreScore:  true,
+			// If PreScore is enabled, the scheduler doesn't execute Score actually.
+			// But, in case PreScore is disabled, Score returns 0 scores to all Nodes.
```

Comment:
I am not sure why this sentence is relevant to this test?

Reply:
echo with the comments here, maybe you should keep the original testcases with `runPreScore` set to `false` and just create another one to validate the `Skip` status.

Reply:
> you should keep the original testcases with runPreScore set to false and just create another one to validate the Skip status.

sg.